data/operators/amenity/police.json

```diff
@@ -755,12 +775,13 @@
       "id": "policianacionaldebolivia-23a32a",
       "locationSet": {"include": ["bo"]},
       "matchNames": [
-        "policia nacional boliviana (pnb)",
-        "policia nacional de bolivia (pnb)"
+        "policia nacional boliviana",
```

Comment:
OH - I left the "(pnb)" part in the `matchName` because these were collected from the OSM planet that way?

https://github.com/osmlab/name-suggestion-index/blob/ab1b26c99b68b249a41edf8e73d07fe7fd86f88a/dist/collected/operators_all.json#L6649-L6652

Sometimes matchName needs to match the weird things that people have previously done.